invenio_administration/menu/menu.py

```diff
@@ -76,7 +76,7 @@ def register_admin_entry(self, current_menu, endpoint):
                 ),
             ),
             order=1,
-            visible_when=lambda: current_user.has_role("admin"),
+            visible_when=lambda: current_user.has_role("administration"),
```

Comment:
I think `administration` is a role that is not created by the system but by the instance managers, meaning that it can change for different instances if I am not mistaken, I guess that we should check for any role and/or user associated with the `administration-access` permission action.

Reply:
I think [here](https://github.com/inveniosoftware/invenio-records-permissions/blob/master/invenio_records_permissions/generators.py#L116-L122) is a good example of what we should look for. If I understand correctly we need to check for 2 possible `Needs` one for roles associated with the action permissions and other for user associated with the action permission.

Reply:
this! 👍🏼